docs/guides/secure-api-auth0.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Securing your API with auth0
+description: Configure an Auth0 application and secure your API with it
+---
+
+## What we'll be doing
+
+In this guide we'll use the Nitric Framework with an [auth0](https://auth0.com/) application to build serverless functions and a secure API.
+
+This example adds secure JWT authentication to an API allowing you to integrate it either with standalone api login, or to integrate it with a frontend application using the clients identity token.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/)
+- The [Nitric CLI](https://nitric.io/docs/installation)
+- An account with [auth0](https://auth0.com/)
+
+## Create an auth0 API
+
+We're assuming you're already using auth0, but if not you can [sign up for free](https://auth0.com/).
+
+Normally when authenticating a frontend application with auth0 you will create a new auth0 application that can be used to log into your app. These log ins will supply you with an auth0 identity token that can be used to request access tokens used to consume your backend APIs.
+
+For the purposes of this guide we are only securing a new API, so we only need to create a new API within auth0.
+
+Log into auth0 and create a new API. If you have an existing API you'd like to use, that's fine, just adapt the next steps to match your existing API.
+
+<img
+  alt="navigate to APIs"
+  src="../../assets/img/guides/auth0/auth0-navigate-apis.png"
+  height="700"
+  />
+
+<img
+  alt="create API"
+  src="../../assets/img/guides/auth0/auth0-create-api.png"
+  height="700"
+  />
+
+> You can enter any identifier you want, this guide will continue using 'testing' in subsequent steps.
+
+Now we're ready to hook up our API for testing with our application. For this guide we'll create a new typescript application using `nitric new`, if you have an application already you want to use, just adapt the below steps to your application.
+
+Before we start getting into the code we'll need a bit more information from auth0.
+
+First we'll need the `audience` for our API, this can be found on the settings tab of your new API.
+
+<img
+  alt="get audience value"
+  src="../../assets/img/guides/auth0/auth0-get-audience.png"
+  height="700"
+  />
+
+Second we'll get the `issuer` for our API, this will be our auth0 environment endpoint and easy way to see this is to navigate the the `Test` tab our our API.
```

Comment:
...endpoint. An easy way to see this is to navigate to the `Test` tab of our API.

Reply:
fixed.